examples/bioasq9b.py

```diff
@@ -0,0 +1,232 @@
+                            "endSection": datasets.Value("string"),
+                            "document": datasets.Value("string"),
+                        }
+                    ),
+                }
+            )
+        # simplified scheam for QA tasks
+        elif self.config.name == "bigbio":
+            features = datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "document_id": datasets.Value("string"),
+                    "question_id": datasets.Value("string"),
+                    "question": datasets.Value("string"),
+                    "type": datasets.Value("string"),
+                    "context": datasets.Value("string"),
+                    "answer": datasets.Sequence(datasets.Value("string")),
+                }
+            )
+
+        return datasets.DatasetInfo(
+            # This is the description that will appear on the datasets page.
+            description=_DESCRIPTION,
+            # This defines the different columns of the dataset and their types
+            features=features,  # Here we define them above because they are different between the two configurations
+            # If there's a common (input, target) tuple from the features,
+            # specify them here. They'll be used if as_supervised=True in
+            # builder.as_dataset.
+            supervised_keys=None,
+            # Homepage of the dataset for documentation
+            homepage=_HOMEPAGE,
+            # License for the dataset if available
+            license=_LICENSE,
+            # Citation for the dataset
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager):
+        """Returns SplitGenerators."""
+        data_dir = dl_manager.download_and_extract(list(_URLs.values()))
+        # BioASQ test data is split into multiple records {9B1_golden.json,...,9B5_golden.json}
+        # combine these files into a single test set 9Bx_golden.json
+        if not os.path.exists(
+            os.path.join(data_dir, "Task9BGoldenEnriched/9Bx_golden.json")
+        ):
+            pass
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                # These kwargs will be passed to _generate_examples
+                gen_kwargs={
+                    "filepath": os.path.join(
+                        data_dir, "BioASQ-training9b/training9b.json"
+                    ),
+                    "split": "train",
+                },
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.TEST,
+                # These kwargs will be passed to _generate_examples
+                gen_kwargs={
+                    "filepath": os.path.join(
+                        data_dir, "BioASQ-training9b/training9b.json"
+                    ),
+                    "split": "test",
+                },
+            ),
+        ]
```

Comment:
we should probably leave some instructions in the hackathon README about what to do if the dataset does not contain a canonical train/val/test split. looks like these files are the same for train and test ... I think we should default to just a `train` set if there is no canonical split.

Reply:
This is just an oversight in this case -- there is a canonical test set. More generally, I think we should be in a setting where there is always a train split defined and (hopefully) a public test or validation set. If not, defaulting to `train` is good with me